lib/scope.js

```diff
@@ -272,22 +274,26 @@ function defineScope(cls, targetClass, name, params, methods, options) {
         f._targetClass = rel && rel.modelTo && rel.modelTo.modelName || i8n.camelize(f._scope.collect);
       }
 
-      f.getAsync = function(condOrRefresh, options, cb) {
+      f.find = function(condOrRefresh, options, cb) {
         if (typeof condOrRefresh === 'function' &&
             options === undefined && cb === undefined) {
-          // customer.orders.getAsync(cb)
           cb = condOrRefresh;
           options = {};
           condOrRefresh = {};
         } else if (typeof options === 'function' && cb === undefined) {
-          // customer.orders.getAsync(condOrRefresh, cb);
           cb = options;
           options = {};
         }
         options = options || {};
         return definition.related(self, f._scope, condOrRefresh, options, cb);
       };
 
+      f.getAsync = function() {
+        deprecated(g.f('Scope method "getAsync()" is deprecated, use "find()" instead.'));
+        return this.find.apply(this, arguments);
+      };
+
+      f.getAsync = f.find; // deprecated alias
```

Comment:
I think you forgot to remove this line, it's overriding the version of `f.getAsync` that's printing a deprecation warning.